### ISSUE TITLE
Water renders beyond render distance with Ocean Physics enabled.

### ISSUE THREAD

**Issue by VOLKOUL** [haubna/PhysicsMod#653]
Minecraft Version: 1.18.2
Physics Mod Version: 1.18.2 v153
Mod Loader: Forge

There is also a ton of optimization mods installed, it is unlikely however that they are the cause of this :P

![Screenshot (518)](https://github.com/haubna/PhysicsMod/assets/97387920/d9bb3806-a79c-4206-ae0b-584fb8358970)
![Screenshot (519)](https://github.com/haubna/PhysicsMod/assets/97387920/c0cdaaef-d764-4f51-844a-3885fd9d4c13)
![Screenshot (522)](https://github.com/haubna/PhysicsMod/assets/97387920/c38094b3-3b88-42b9-bdfb-9f820f1c2ca5)
![Screenshot (517)](https://github.com/haubna/PhysicsMod/assets/97387920/6c901d4b-ef59-48c9-bcbb-74f0a84be5a0)

**Comment by VOLKOUL** [comment]
It's more than 1 chunk aparently
![Screenshot (524)](https://github.com/haubna/PhysicsMod/assets/97387920/dc5b5a01-a5ce-4989-a890-3c71eedc192f)
![Screenshot (525)](https://github.com/haubna/PhysicsMod/assets/97387920/3ba5f647-b2b8-4c78-88e5-f8d17630d932)
![Screenshot (523)](https://github.com/haubna/PhysicsMod/assets/97387920/e643ffce-4878-4241-96ed-6d0b57cd7595)

**Comment by haubna** [comment]
Yup that can happen since the chunks are actually loaded in memory. Usually fog hides it but I guess you disabled it somehow :P

**Comment by MarkinSCode** [comment]
This problem appears when using the "Better Fps - Render Distance" mod, which reduces the vertical render distance, but PhysicsMod does not seem to take this into account.

**Comment by VOLKOUL** [comment]
I do not use the "Better Fps - Render Distance mod"
The fog is off because of Rubidium Extra :D
